src/scss/components/settings/subtitlesettings/_subtitle-settings-reset-button.scss

```diff
@@ -1,12 +1,13 @@
 @import '../../../variables';
 
-.#{$prefix}-ui-subtitlesettingsresetbutton {
+.#{$prefix}-ui-textbutton {
```

Comment:
Couple of things:
- I see the need for a change since there are multiple different use cases now for this button. However, the new name `textbutton` would indicate that it's some sort of generic button which can be used everywhere. My suggestion would be to tailor this to something like a `settingsPanelPageTitleButton` or similar.
- please also reflect the component name in the file name
- if no longer explicitly for subtitle settings, please also move it out of the folder.

Reply:
I renamed the style to `ui-settings-panel-navigation-text-button` and moved things around so that it isn't subtitle specific: 
https://github.com/bitmovin/bitmovin-player-ui/commit/9b99e4389e0010fac74ae3b285c278f5106464f0